client/app/achievements/achievements.html

```diff
@@ -11,34 +11,11 @@
 
 <div class="container">
   <div class="row text-area">
-    <div class="col-md-8">
```

Comment:
oh nice. Just read from here:  http://stackoverflow.com/questions/20171408/how-do-i-change-bootstrap-3-column-order-on-mobile-layout `You cannot change the order of columns in smaller screens but you can do that in large screens` guess it's changed.